include/dlaf/blas/tile.h

```diff
@@ -81,6 +81,23 @@ inline void extendROCBlasWorkspace(cublasHandle_t handle,
   template <typename T>               \
   struct Name
 
+#ifdef DLAF_WITH_HIP
+    // rocblas functions needs to be lowercase
```

Comment:
```suggestion
    // rocblas functions are lowercase
```
but I also think the comment doesn't add much value and could be removed?